packages/sit-onyx/src/components/OnyxSeparator/OnyxSeparator.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="onyx-separator"></div>
```

Comment:
```suggestion
  <!-- TODO: Add aria context: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/separator_role -->
  <div class="onyx-separator"></div>
```

Reply:
Nice I didn't know about that role :) I added it so we don't need the TODO